README.md

```diff
@@ -4,7 +4,7 @@ Jaeger-Swift is a distributed tracing client for Uber's [Jaeger](https://www.jae
 
 ## Installation
 
-This section has quick setup steps for getting a simple proof of concept up and running. For a more comprehensive understanding of this project, Check out our [API reference](). A complete Opentracing solution consists of a client, an agent and a collector. The following steps will show you how to configure this project to report to a locally hosted Jaeger collector.
+This section has quick setup steps for getting a simple proof of concept up and running. For a more comprehensive understanding of this project, Check out our [API reference](https://wayfair.github.io/jaeger-swift/docsets/jaeger-swift.xml). A complete Opentracing solution consists of a client, an agent and a collector. The following steps will show you how to configure this project to report to a locally hosted Jaeger collector.
```

Comment:
I'm sorry, I think I misunderstood what you meant by linking to the docset. The reference URL should link to the site itself, at `https://wayfair.github.io/jaeger-swift`